supabase/seed/seed.mts

```diff
@@ -72,13 +72,23 @@ async function generateSeed(): Promise<void> {
                 generateFamily(
                     { min: 1, max: 8 },
                     {
-                        age: (ctx) => copycat.int(ctx.seed, { min: 0, max: 100 }),
+                        birthYear: (ctx) => copycat.int(ctx.seed, { min: 1901, max: 2024 }),
+                        birthMonth: null,
                         gender: (ctx) => copycat.oneOf(ctx.seed, genders),
                     }
                 ),
         })
     );
 
+    await seed.families(
+        (generate) =>
+            generate(100, {
+                birthYear: 2024,
+                birthMonth: (ctx) => copycat.int(ctx.seed, { min: 1, max: 5 }),
```

Comment:
Why only up to 5?

Reply:
Only up to May (current month) otherwise the babies are not going to be born yet.